### ISSUE TITLE
Improper Version Number in release 3.3.13. postfixadmin-3.3.13 showing $CONF['version'] = '3.3.11'; in config.inc.php

### ISSUE THREAD

**Issue by Danrancan** [postfixadmin/postfixadmin#854]
> Look for the line **$CONF['version']** in config.inc.php file of your postfixadmin installation. Example; $CONF['version'] = '3.3.11';

Here is why I am getting confused...
In [postfixadmin-3.3.11](https://github.com/postfixadmin/postfixadmin/releases/tag/postfixadmin-3.3.11)`/config.inc.php`, the $CONF['version'] = '`3.3.11`';

This seems correct.

and in [postfixadmin-3.3.13](https://github.com/postfixadmin/postfixadmin/releases/tag/postfixadmin-3.3.13)`/config.inc.php` the $CONF['version'] **ALSO** = '`3.3.11`';

This does not seem correct. Shouldn't it be $CONF['version'] ALSO = '`3.3.13`'; ?

_Originally posted by @Danrancan in https://github.com/postfixadmin/postfixadmin/issues/852#issuecomment-2323613712_

**Comment by DavidGoodwin** [comment]
Yes, unfortunately some muppet 🙄 (me) hadn't updated the release number in the config file ... hence #845, #778, #740 and presumably #694 

Sorry!
